storage/feed.go

```diff
@@ -79,6 +79,34 @@ type StartStoreEvent struct {
 	StoreID proto.StoreID
 }
 
+// StoreStatusEvent contains the current descriptor for the given store.
+//
+// Because the descriptor contains information that cannot currently be computed
+// from other events, this event should be periodically broadcast by the store
+// independently of other operations.
+type StoreStatusEvent struct {
+	StoreID proto.StoreID
```

Comment:
Seemed better to me to have the redundant information to keep consistency with the rest of the events.